openmetadata-ui/src/main/resources/ui/src/pages/CustomPropertiesPageV1/CustomPropertiesPageV1.tsx

```diff
@@ -51,8 +54,7 @@ const CustomEntityDetailV1 = () => {
   const [selectedEntityTypeDetail, setSelectedEntityTypeDetail] =
     useState<Type>({} as Type);
 
-  const tabAttributePath =
-    customAttributesPath[tab as keyof typeof customAttributesPath];
+  const tabAttributePath = ENTITY_PATH[tab.toLowerCase() as EntityType];
```

Comment:
@chirag-madlani `ENTITY_PATH`  is expecting key in small case and one value `tab` having value in uppercase. Since it's only affecting here that why `toLowerCase`  is used.